include/nanorange/views/subrange.hpp

```diff
@@ -197,7 +197,7 @@ class subrange : public view_interface<subrange<I, S, K>> {
               typename = std::enable_if_t<
                   detail::convertible_to_non_slicing<II, I> &&
                   KK == subrange_kind::sized>>
-    constexpr subrange(II i, S s, iter_difference_t<I> n)
+    constexpr subrange(II i, S s, std::make_unsigned_t<iter_difference_t<I>> n)
```

Comment:
The standard says `make-unsigned-like-t`, not `std::make_unsigned_t`. I'm not 100% sure this is correct.

https://eel.is/c++draft/range.subrange